### ISSUE TITLE
Change searchbox shortcut to ctrl-k/command-k, and make it visible

### ISSUE THREAD

**Issue by adamchainz** [django/djangoproject.com#1112]
One can press the / key to focus the search box, but this is a secret feature. We should surface this - perhaps as easy as changing the search box's placeholder text, or by adding a small `<kbd>` element as can be seen on [tailwind](https://tailwindcss.com/) and [bootstrap](https://getbootstrap.com/docs/5.1/getting-started/introduction/).

**Comment by emorozov** [comment]
I find this feature very annoying. Most of the time I use / key to start search on the page (default Tridactyl keybinding) and due to docs.djangoproject.com intercepting it in a way that is impossible to override (I've tried but failed) I have to press several keys every time when I search something.

Besides, the search feature on docs.djangoproject.com is pretty much useless - I don't use it because it always returns ton of unwanted results, e.g. when you search a name of some utility function, it will return all mentions of it for all the versions of Django (3.2, 3.1, 3.0, 2.2, etc). It's way easier to just type `utility_func site:docs.djangoproject.com` in the search engine, or simply `utility_func django` rather than use built-in search.

And that inconvenient search feature binding shadows an actually useful binding...

(I just came here to see if someone already suggested to remove the annoying / binding or at least suggest a way to override it for users who find it annoying like me)

**Comment by adamchainz** [comment]
We could consider changing the key binding, yes. "/" is "quick find" in firefox, so it's an annoying collision.

I notice that some sites e.g. Tailwind use ctrl-k/command-k, which is probably a good sign that it doesn't conflict with anything.

**Comment by jnns** [comment]
Coincidentally, <kbd>CTRL</kbd>+<kbd>k</kbd> is a keyboard shortcut for focusing the search/address bar in both [Firefox](https://support.mozilla.org/en-US/kb/keyboard-shortcuts-perform-firefox-tasks-quickly#w_search) and [Chrome](https://support.google.com/chrome/answer/157179?hl=en&co=GENIE.Platform%253DDesktop#zippy=%2Caddress-bar-shortcuts). Maybe that's why Tailwind chose it in their docs to focus their own search prompt. 

It wouldn't be too bad to chose <kbd>CTRL</kbd>+<kbd>k</kbd> because Firefox also has <kbd>CTRL</kbd>+<kbd>j</kbd> as an alternative. Chrome has <kbd>CTRL</kbd>+<kbd>e</kbd> as an alternative.

**Comment by pauloxnet** [comment]
I don't think this is stale. We only need a PR to start working on it.

**Comment by adamchainz** [comment]
Just noticed that MDN also uses Command-K / Ctrl-K. I think we should copy this convention.

**Comment by megaden** [comment]
Hey guys, I prepared [a PR #1236](https://github.com/django/djangoproject.com/pull/1236) with the fix for this one, PLMK what do you think.
